TezosKit/Models/TezosBalance.swift

```diff
@@ -86,7 +86,8 @@ public struct TezosBalance {
       paddedBalance.index(paddedBalance.startIndex,
                           offsetBy: paddedBalance.count - decimalDigitCount)
 
-    let integerString = paddedBalance[paddedBalance.startIndex ..< integerDigitEndIndex].count > 0 ? paddedBalance[paddedBalance.startIndex ..< integerDigitEndIndex] : "0"
+    let integerString = paddedBalance[paddedBalance.startIndex ..< integerDigitEndIndex].isEmpty ?
```

Comment:
you are generating the range twice here, why not pull it into a local var?

Reply:
Done.